content/docs/command-reference/remote/modify.md

```diff
@@ -375,24 +389,24 @@ authentication method:
   $ dvc remote modify --local myremote client_secret 'client-secret'
   ```
 
-- `account_name` - storage account name. May work by itself (via
-  [anonymous auth](https://docs.microsoft.com/en-us/azure/storage/blobs/anonymous-read-access-configure))
-  or along with either `account_key` or `sas_token` along with this):
+* `account_key` - storage account key:
```

Comment:
Should mention about default credentials here too instead anonymous auth?

> p.s. I rescued the anon auth link for step 4 above.

Reply:
We already mention it in the step 5, so I don't think that is necessary.